apps/web/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useState } from 'react';
+import './SearchBar.scss';
+
+export interface Props {
+  /** What happens when we use a searchstring to search for results? */
+  handleSearch: (searchString: string) => void;
+  /**  An optional custom test ID for the component.  */
+  customTestId?: string;
+}
+
+// need to add jsdoc comment
+const SearchBar = ({
+  handleSearch,
+  customTestId = 'searchbar',
+}: Props): React.ReactElement => {
+  /** Tracks the string value within input */
+  const [searchString, setSearchString] = useState<string>('');
```

Comment:
Why are we storing `searchString` in local state? Think of `useState` kind of like `useEffect` - always have a good reason for using it, as it can make debugging or understanding the life cycle of a given component more difficult. 

You say that it "track the string value within input" - doesn't an `input` already do that?